### PR TITLE
mqtt unstable diconnects and reconnects every minute

### PR DESCRIPTION
**Pull request by manivalaguru**

### CODE REVIEW

**Comment by btsimonh** [comment]
hi @manivalaguru 
could you rebase this on top of current main, and highlight just the changes which may fix mqtt?
At the moment, most of the changes revert a lot of current updates, and it's very hard to follow 'the good stuff' :(.
Simon

**Comment by btsimonh** [comment]
hi @manivalaguru 
please could you try release 70?
For N and T this resolves MQTT issues at a base level, ensuring we call functions in the correct context, and don't process input data within the tcpip_thread...
br,
Simon

**Comment by openshwprojects** [comment]
It should be totally fixed now.
